Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -955,6 +955,7 @@ open class ALKConversationViewController: ALKBaseViewController, Localizable {
 
     /// Call this method after proper viewModel initialization
     open func refreshViewController() {
+        KMTextToSpeechHandler.shared.resetSynthesizer()
```

Comment:
We should also stop it when they go back from this VC.

Reply:
yeah it will stop the voice once we tap the back button on VC.done